apps/examples/webserver_test/results.md

```diff
@@ -0,0 +1,61 @@
+# Results
+
+### Case 1: HTTP, keep-alive OFF , Chunk-transfer OFF :
+
+| Test_Type | Total_checks | Pass | Fail |
+| ------|------ | ------|------ |
```

Comment:
1. It will be better is there is sync such as 

 | Test_Cmd | Total_Iterations  | Pass | Fail  |
 | -----------|-------------------|------|----- |

2. Can you write TASH command for each result also.